lib/config/file.go

```diff
@@ -42,13 +42,13 @@ func NewNamespacesFromDir(nsdir string) (*NamespaceDir, error) {
 		nspath: make(map[string]string),
 	}
 
-	yamlFiles, err := listAllYamlFiles(c.dir)
+	nFiles, err := listAllFiles(c.dir, ".toml")
```

Comment:
Why change to toml files this time? Is it required by the operator?

Reply:
Kind of. Operator has built-in support for json and toml but not yaml.
Anyway, dynamic namespace will not be done by using configuration file. Also this is duplicated with `weirctl`.
It is not used at all, I removed this.